packages/kbn-lens-embeddable-utils/config_builder/utils.ts

```diff
@@ -75,6 +77,22 @@ export const getAdhocDataviews = (dataviews: Record<string, DataView>) => {
   return adHocDataViews;
 };
 
+export function isFormulaValue(value: unknown): value is FormulaValueConfig {
+  if (value && typeof value === 'object' && 'formula' in value) {
+    return true;
+  }
+  return false;
```

Comment:
I think this is the same, right?
```suggestion
 return value && typeof value === 'object' && 'formula' in value;
```

Reply:
I think unit testing this function and the one below would be nice.

Reply:
Tests are passing through them despite not unit testing them. There is not much to validate there.